data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1226 @@
+			`	Before you leave, you watch as they lift their staff weapon and flip it around. An explosive cloud of smoke follows, and a sizable crater appears just barely avoiding your ship on the ground. It seems that you're no longer welcome here, though, the fact that they intentionally missed such a large target might have other implications. Regardless, it could have been you on the other end of that blast at any point during this journey.`
+			`	Despite the aggressive action, you are not followed as you leave the planet.`
+				flee
+
+
+			label "no more gegno"
+			`	You ultimately decide that pursuing this ruse with the Gegno is no longer worth the effort. Even if you did meet up with them, or ended up in some sort of debate with them, it would likely take an excessive amount of effort in order to get any amount of progress, if it's even possible. Perhaps even landing here was a mistake, as you could never guess what might happen if you continued this endeavor, given you are all alone on the home world of a species that has shown itself to be rather xenophobic, even to the Quarg.`
+			`	As you prepare to depart, the Gegno, who was watching you all this time with a blank stare, lifts their staff weapon and flips it around. An explosive cloud of smoke follows, and a sizable crater appears just barely avoiding your ship on the ground. It seems they are, at the very least, unhappy with your decision to leave, which is a little odd considering they didn't want anything to do with you before. Though, the fact they intentionally missed such a large target might have other implications. Regardless, it could have been you on the other end of that blast.`
+			`	Despite the aggressive action, you are not followed as you leave the planet.`
+				flee
+
+
+
+mission "Gegno Asteroid Mining Prologue"
+	landing
+	name "Mine 5 tons of Iron"
+	description "Mine and return 5 tons of Iron to <origin> in the Oosuoro system."
+	source "Tschyss"
+	to offer
+		has "Acquiescence: offered"
+		not "Gegno Genocide Defense: offered"
+	on offer
+		conversation
+			branch "returned"
+				has "Gegno Asteroid Mining Prologue: deferred"
+
+			`You arrive at your destination: a massive space station. Instead of standing directly behind you in your cockpit this time, the Gegno ended up taking a seat next to one of your open bunks, not moving a muscle up until now. Despite the grandiose nature of the Gegno's one and only space station, it seems very old and inactive. As you are escorted around the spacious interior, of which a majority is only a skeletal frame of pipes, bars, and supportive structure, you see a noticeable lack of personnel and running machinery. Some sections aren't even properly pressurized, but thankfully, your spacesuit helps with that. Occasionally, you come across some workers, but they appear very lethargic, or maybe just unmotivated.`
```

Comment:
```suggestion
			`You arrive at your destination: a massive space station. Instead of standing directly behind you in your cockpit this time, the Gegno ended up taking a seat next to one of your open bunks, and didn't move a muscle up until now. Despite the grandiose nature of the Gegno's one and only space station, it seems very old and inactive. As you are escorted around the spacious interior, of which a majority is only a skeletal frame of pipes, bars, and supportive structure, you see a noticeable lack of personnel and running machinery. Some sections aren't even properly pressurized, but, thankfully, your spacesuit helps with that. Occasionally, you come across some workers, but they appear very lethargic, or maybe just unmotivated.`
```